File: python/S1_BASIC/L4_BASICMATHS/countdigits.py
class Solution:
    def __init__(self,num):
        self.num=num
    def digitcount(self):
        count=0
        temp=self.num
        while temp>0:
            count+=1
            temp=temp//10
        return count
if __name__=="__main__":
    n=int(input("enter number :"))
    s=Solution(n)
    print(s.digitcount())

File: JAVA/S1_BASIC/L4_BASICMATHS/countdigits.java

class solution{
    int countdigit(int num){
        int count=0;
        while(num>0){
            count++;
            num/=10;
        }
        return count;
    }
}
public class countdigits{
    public static void main(String[] args){
        solution s=new solution();
        System.out.println(s.countdigit(2345));
    }

}